### PR TITLE
feat: added text utilities to CL

### PR DESCRIPTION
**Pull request by ooreally**
feat: added text utilities to the CL
Here is the ss 
image

### CODE REVIEW

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

File: components/text/text-util.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.sm{
+    font-size: 0.8rem;
+}
+
+.center{
+    display:flex;
+    justify-content: center;
+
```
Comment (sharmayatendra) [inline components/text/text-util.css:29]: try to use proper spacing

**Review by sharmayatendra — APPROVED** [review]
looks good can look upon suggested changes

File: components/text/text-util.css (hunk 1 above)
Comment (ooreally) [reply components/text/text-util.css:29]: ok

**Pull request merged**